src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/TestData/Expected/Hl7v2/ADT_A01/ADT04-28-expected.json

```diff
@@ -226,6 +226,10 @@
       "resource": {
         "resourceType": "Provenance",
         "id": "a81dc37f-d7f3-c46a-00b1-4ff08fa99061",
+        "text": {
+          "status": "generated",
+          "div": "<div xmlns=\"http://www.w3.org/1999/xhtml\"><p><span>Resource bundle generated on 2021-04-09T05:48:16.596Z using Microsoft FHIR Converter.</span><span>Template Version: 3.5.1.</span><span>Template URL: https://github.com/microsoft/FHIR-Converter/releases/download/v3.5.1/Hl7v2DefaultTemplates.tar.gz.</span><span>Root template: ADT_A01.</span></p><p>Disclaimer: Real world HL7 messages vary in richness and level of conformance with the spec. The output of converter depends on the templates as well as the quality and richness of input messages. Therefore, it is important that you review and validate the Converter output before using those in production.</p></div>"
```

Comment:
Shall we wrap the version information to something like __ENGINE_VERSION__ in the test code like you do in the regex below so that we don't have to change all json files for each upgrade.
```csharp
            var regex = new Regex(@"Resource bundle generated on .* using Microsoft FHIR Converter");
```

Reply:
You mean we replace the version number `3.5.1` with `TEMPLATE_VERSION` literally in test JSON file so that we don't need to change the number each time we update the templates?

Reply:
correct

Reply:
That's a good idea, I'll update the test resources.